### PR TITLE
updating enterprise documentation with additional clarity

### PR DESCRIPTION
**Pull request by codyde**
Added additional content to the Enterprise sections for several features based on feedback. Also included additional learn guides were appropriate.

### CODE REVIEW

File: website/source/docs/enterprise/backups/index.html.md [hunk 1]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
-Consul's core snapshot functionality allows operators to save and restore the state of
-the Consul servers for disaster recovery. Snapshots are atomic and
-point-in-time, and include key/value entries, service catalog, prepared
-queries, sessions, and ACLs.
 
-[Consul Enterprise](https://www.hashicorp.com/consul.html) provides a [highly
-available service](/docs/commands/snapshot/agent.html) that
-integrates with the snapshot API to automatically manage taking snapshots,
-perform rotation and send backup files offsite to GCP, Amazon S3, or another S3-compatible endpoint.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) enables customers to run
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) enables you to run
```

File: website/source/docs/enterprise/backups/index.html.md [hunk 2]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
+the snapshot agent within their environment as a service (Systemd as an example)
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
the snapshot agent within your environment as a service (Systemd as an example)
```

File: website/source/docs/enterprise/backups/index.html.md [hunk 3]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
+or scheduled through other means. Once running, this service operates as a highly
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
or scheduled through other means. Once running, the snapshot agent service operates as a highly 
```

File: website/source/docs/enterprise/backups/index.html.md [hunk 4]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
+[Consul Enterprise](https://www.hashicorp.com/consul.html) enables customers to run
+the snapshot agent within their environment as a service (Systemd as an example)
+or scheduled through other means. Once running, this service operates as a highly 
+available process that integrates with the snapshot API to automatically manage 
+taking snapshots, backup rotation, and sending backup files offsite to Amazon S3 
+(or another S3-compatible endpoint), Google Cloud Storage, or Azure Blob Storage.
+
+This capability provides an enterprise solution for backup and restoring the state of Consul servers 
+within an environment in an automated manner. These Snapshots are atomic and point-in-time. Consul 
+cluster backups include:
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
datacenter backups include: 
```

Comment (kaitlincarter-hc) [reply website/source/docs/enterprise/backups/index.html.md]: I'd add language around "include but is not limited to"

File: website/source/docs/enterprise/backups/index.html.md [hunk 5]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
+cluster backups include: 
+
+- Key/Value Store Entries
+- Service Catalog Registrations 
+- Prepared Queries 
+- Sessions
+- Access Control Lists (ACLs)
+- Namespaces
+
+For more experience leveraging Consuls snapshot functionality, we suggest you look through our HashiCorp
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
For more experience leveraging Consul's snapshot functionality, we suggest you look through our HashiCorp 
```

File: website/source/docs/enterprise/backups/index.html.md [hunk 6]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
+- Key/Value Store Entries
+- Service Catalog Registrations 
+- Prepared Queries 
+- Sessions
+- Access Control Lists (ACLs)
+- Namespaces
+
+For more experience leveraging Consuls snapshot functionality, we suggest you look through our HashiCorp 
+Learn guide for [Datacenter Backups in Consul](https://learn.hashicorp.com/consul/datacenter-deploy/backup). 
+For detailed configuration information on configuring this functional, review the
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
For detailed configuration information on configuring the Consul Enterprise's snapshot agent, review the 
```

File: website/source/docs/enterprise/index.html.md [hunk 7]
```diff
@@ -8,19 +8,19 @@ description: |-
 
 # Consul Enterprise
 
-Consul Enterprise simplifies operations by automating workflows. It adds support
-for microservices deployments across complex network topologies. It also
-increases both scalability and resilience. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing)
-below.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) focuses on the concerns of operating Consul at
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/index.html.md]:
```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) features ease the complexity of operating Consul at 
```
Or similar..

File: website/source/docs/enterprise/index.html.md [hunk 8]
```diff
@@ -8,19 +8,19 @@ description: |-
 
 # Consul Enterprise
 
-Consul Enterprise simplifies operations by automating workflows. It adds support
-for microservices deployments across complex network topologies. It also
-increases both scalability and resilience. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing)
-below.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) focuses on the concerns of operating Consul at 
+organizational scale by automating common operator workflows. It introduces capabilities for expanding 
+performance scalability, resiliency and cluster lifecycle. It also adds support for application and service
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/index.html.md]:
```suggestion
performance scalability, resiliency, and cluster lifecycle. It also adds support for application and service 
```

File: website/source/docs/enterprise/index.html.md [hunk 9]
```diff
@@ -8,19 +8,19 @@ description: |-
...
+[Consul Enterprise](https://www.hashicorp.com/consul.html) focuses on the concerns of operating Consul at 
+organizational scale by automating common operator workflows. It introduces capabilities for expanding 
+performance scalability, resiliency and cluster lifecycle. It also adds support for application and service 
+architectures across complex network topologies. If you have already purchased Consul Enterprise, please 
+see the [licensing section](#licensing) below.
 
 Features include:
 
 - [Automated Backups](/docs/enterprise/backups/index.html)
 - [Automated Upgrades](/docs/enterprise/upgrades/index.html)
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/index.html.md]: @codyde Do you have an asana to track upcoming changes? @danielehc is working on new guides for Redundancy zones and automated backups.

File: website/source/docs/enterprise/index.html.md [hunk 10]
```diff
@@ -8,19 +8,19 @@ description: |-
...
 - [Enhanced Read Scalability](/docs/enterprise/read-scale/index.html)
 - [Redundancy Zones](/docs/enterprise/redundancy/index.html)
-- [Advanced Federation for Complex Network
-  Topologies](/docs/enterprise/federation/index.html)
+- [Advanced Federation for Complex Network Topologies](/docs/enterprise/federation/index.html)
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/index.html.md]: I like that you've specified the use case. We'd discussed a table before, did that not make the cut?

File: website/source/docs/enterprise/namespaces/index.html.md [hunk 11]
```diff
@@ -13,7 +13,7 @@ can be isolated from each other with the use of Namespaces. Namespaces help redu
 by removing restrictions around uniqueness of resource names across distinct teams, and enable operators 
 to provide self-service through delegation of administrative privileges.
 
-For more information on how to use namespaces with Consul Enterprise please review the following Learn Guides:
+For more information on how to use namespaces with Consul Enterprise please review the following HashiCorp Learn Guides:
 
 - [Register and Discover Services within Namespaces](https://learn.hashicorp.com/consul/namespaces/discovery-namespaces) - Register multiple services within different Namespaces in Consul
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/namespaces/index.html.md]:
```suggestion
- [Register and Discover Services within Namespaces](https://learn.hashicorp.com/consul/namespaces/discovery-namespaces) - Register multiple services within different Namespaces in Consul.
```

File: website/source/docs/enterprise/namespaces/index.html.md [hunk 12]
```diff
@@ -13,7 +13,7 @@ can be isolated from each other with the use of Namespaces. Namespaces help redu
...
 - [Setup Secure Namespaces](https://learn.hashicorp.com/consul/namespaces/secure-namespaces) - Secure resources within a namespace and delegate Namespace ACL rights via ACL tokens
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/namespaces/index.html.md]:
```suggestion
- [Setup Secure Namespaces](https://learn.hashicorp.com/consul/namespaces/secure-namespaces) - Secure resources within a namespace and delegate Namespace ACL rights via ACL tokens.
```

File: website/source/docs/enterprise/namespaces/index.html.md (hunk 11 above)
Comment (kaitlincarter-hc) [reply website/source/docs/enterprise/namespaces/index.html.md]: Can you also lower case https://github.com/hashicorp/consul/pull/7495/files#diff-a7c1e35dcabd95e1f65d084fe10ad7f4R12 ?

File: website/source/docs/enterprise/backups/index.html.md [hunk 13]
```diff
@@ -3,17 +3,31 @@ layout: "docs"
...
-perform rotation and send backup files offsite to GCP, Amazon S3, or another S3-compatible endpoint.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) enables you to run
+the snapshot agent within your environment as a service (Systemd as an example)
+or scheduled through other means. Once running, the snapshot agent service operates as a highly 
+available process that integrates with the snapshot API to automatically manage 
+taking snapshots, backup rotation, and sending backup files offsite to Amazon S3 
+(or another S3-compatible endpoint), Google Cloud Storage, or Azure Blob Storage.
+
+This capability provides an enterprise solution for backup and restoring the state of Consul servers 
+within an environment in an automated manner. These Snapshots are atomic and point-in-time. Consul
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/backups/index.html.md]:
```suggestion
within an environment in an automated manner. These snapshots are atomic and point-in-time. Consul 
```

File: website/source/docs/enterprise/network-segments/index.html.md [hunk 14]
```diff
@@ -7,16 +7,16 @@ description: |-
...
 Consul Network Segments enables operators to create separate LAN gossip segments
 in one Consul cluster. Agents in a segment are only able to join and communicate
-with other agents in its network segment. This functionality is useful for
+with other agents in it's network segment. This functionality is useful for
 clusters that have multiple tenants that should not be able to communicate
 with each other.
 
-To get started with Network Segments,
-[read the guide](https://learn.hashicorp.com/consul/day-2-operations/network-segments).
+To get started with Network Segments you can review the guide on HashiCorp Learn for
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/network-segments/index.html.md]:
```suggestion
To get started with network segments you can review the guide on HashiCorp Learn for
```

File: website/source/docs/enterprise/network-segments/index.html.md [hunk 15]
```diff
@@ -38,7 +38,8 @@ cluster each set per "datacenter". These Consul servers are federated together
 over the WAN. Consul clients make use of resources in federated clusters by
 forwarding RPCs through the Consul servers in their local cluster, but they
 never interact with remote Consul servers directly. There are currently two
-inter-cluster network models: [WAN Gossip (OSS)](https://learn.hashicorp.com/consul/security-networking/datacenters)
+inter-cluster network models which can be viewed on HashiCorp Learn: 
+[WAN Gossip (OSS)](https://learn.hashicorp.com/consul/security-networking/datacenters)
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/network-segments/index.html.md]:
```suggestion
[WAN gossip (OSS)](https://learn.hashicorp.com/consul/security-networking/datacenters)
```

File: website/source/docs/enterprise/read-scale/index.html.md [hunk 16]
```diff
@@ -3,14 +3,17 @@ layout: "docs"
 page_title: "Consul Enterprise Enhanced Read Scalability"
 sidebar_current: "docs-enterprise-read-scale"
 description: |-
-  Consul Enterprise supports increased read scalability without impacting write latency.
+  Consul Enterprise supports increased read scalability without impacting write latency by introducing
+  non-voting servers.
 ---
 
-# Consul Enterprise Enhanced Read Scalability
+# Enhanced Read Scalability / Non-Voting Servers
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/read-scale/index.html.md]:
```suggestion
# Enhanced Read Scalability with Non-Voting Servers
```

File: website/source/docs/enterprise/read-scale/index.html.md [hunk 17]
```diff
@@ -3,14 +3,17 @@ layout: "docs"
...
 
-In [Consul Enterprise](https://www.hashicorp.com/consul.html), servers can be
-explicitly marked as non-voters. Non-voters will receive the replication stream
-but will not take part in quorum (required by the leader before log entries can
-be committed). Adding explicit non-voters will scale
-reads
-without impacting write latency.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) provides the ability to scale Consul server clusters
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/read-scale/index.html.md]:
```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) provides the ability to scale Consul servers
```

File: website/source/docs/enterprise/read-scale/index.html.md [hunk 18]
```diff
@@ -3,14 +3,17 @@ layout: "docs"
...
+to include voting and non-voting servers. Non-voting servers still receive data from the cluster replication
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/read-scale/index.html.md]:
```suggestion
to include voting and non-voting servers. Non-voting servers still receive data from the cluster replication,
```

File: website/source/docs/enterprise/read-scale/index.html.md [hunk 19]
```diff
@@ -3,14 +3,17 @@ layout: "docs"
...
+however do not take part in quorum election operations. Expanding your Consul cluster in this way can scale
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/read-scale/index.html.md]:
```suggestion
however, they do not take part in quorum election operations. Expanding your Consul cluster in this way can scale
```

File: website/source/docs/enterprise/read-scale/index.html.md [hunk 20]
```diff
@@ -3,14 +3,17 @@ layout: "docs"
...
+reads without impacting write latency. 
+
+For more details, review the [Consul Server configuration](https://www.consul.io/docs/agent/options.html)
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/read-scale/index.html.md]:
```suggestion
For more details, review the [Consul server configuration](https://www.consul.io/docs/agent/options.html) 
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 21]
```diff
@@ -6,14 +6,22 @@ description: |-
...
-[Consul Enterprise](https://www.hashicorp.com/consul.html) [redundancy
-zones](https://learn.hashicorp.com/consul/day-2-operations/autopilot#redundancy-zones) make
-it possible to have more servers than availability zones. For example, in an
-environment with three availability zones it's now possible to run one voter and
-one non-voter in each availability zone, for a total of six servers. If an
-availability zone is completely lost, only one voter will be lost, so the
-cluster remains available. If a voter is lost in an availability zone, Autopilot
-will promote the non-voter to voter automatically, putting the hot standby
-server into service quickly.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) Redundancy Zones provide
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) redundancy zones provide 
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 22]
```diff
@@ -6,14 +6,22 @@ description: |-
...
-it possible to have more servers than availability zones. For example, in an
-environment with three availability zones it's now possible to run one voter and
-one non-voter in each availability zone, for a total of six servers. If an
-availability zone is completely lost, only one voter will be lost, so the
-cluster remains available. If a voter is lost in an availability zone, Autopilot
-will promote the non-voter to voter automatically, putting the hot standby
-server into service quickly.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) Redundancy Zones provide 
+both scaling and resiliancy benefits by enabling the deployment of non-voting
+servers alongside voting servers on a per Availability Zone basis.
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
servers alongside voting servers on a per availability zone basis.  
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 23]
```diff
@@ -6,14 +6,22 @@ description: |-
...
-one non-voter in each availability zone, for a total of six servers. If an
-availability zone is completely lost, only one voter will be lost, so the
-cluster remains available. If a voter is lost in an availability zone, Autopilot
-will promote the non-voter to voter automatically, putting the hot standby
-server into service quickly.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) Redundancy Zones provide 
+both scaling and resiliancy benefits by enabling the deployment of non-voting
+servers alongside voting servers on a per Availability Zone basis.  
+
+When using Redunancy Zones, if an operator chooses to deploy Consul across 3 Availability Zones, they
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
When using redundancy zones, if an operator chooses to deploy Consul across 3 availability zones, they 
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 24]
```diff
@@ -6,14 +6,22 @@ description: |-
...
-cluster remains available. If a voter is lost in an availability zone, Autopilot
-will promote the non-voter to voter automatically, putting the hot standby
-server into service quickly.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) Redundancy Zones provide 
+both scaling and resiliancy benefits by enabling the deployment of non-voting
+servers alongside voting servers on a per Availability Zone basis.  
+
+When using Redunancy Zones, if an operator chooses to deploy Consul across 3 Availability Zones, they 
+could have 2 (or more) servers (1 voting/1 non-voting) in each zone. In the event that a voting 
+member in an availability zone fails, the Redundancy Zone configuration would automatically
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
member in an availability zone fails, the redundancy zone configuration would automatically 
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 25]
```diff
@@ -6,14 +6,22 @@ description: |-
...
-will promote the non-voter to voter automatically, putting the hot standby
-server into service quickly.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) Redundancy Zones provide 
+both scaling and resiliancy benefits by enabling the deployment of non-voting
+servers alongside voting servers on a per Availability Zone basis.  
+
+When using Redunancy Zones, if an operator chooses to deploy Consul across 3 Availability Zones, they 
+could have 2 (or more) servers (1 voting/1 non-voting) in each zone. In the event that a voting 
+member in an availability zone fails, the Redundancy Zone configuration would automatically 
+promote the non-voting member to a voting member. In the event that an entire Availability
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
promote the non-voting member to a voting member. In the event that an entire availability 
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 26]
```diff
@@ -6,14 +6,22 @@ description: |-
...
-server into service quickly.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) Redundancy Zones provide 
+both scaling and resiliancy benefits by enabling the deployment of non-voting
+servers alongside voting servers on a per Availability Zone basis.  
+
+When using Redunancy Zones, if an operator chooses to deploy Consul across 3 Availability Zones, they 
+could have 2 (or more) servers (1 voting/1 non-voting) in each zone. In the event that a voting 
+member in an availability zone fails, the Redundancy Zone configuration would automatically 
+promote the non-voting member to a voting member. In the event that an entire Availability 
+Zone was lost, a non-voting member in one of the existing availability zones would promote
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
zone was lost, a non-voting member in one of the existing availability zones would promote 
```

File: website/source/docs/enterprise/redundancy/index.html.md [hunk 27]
```diff
@@ -6,14 +6,22 @@ description: |-
...
+servers alongside voting servers on a per Availability Zone basis.  
+
+When using Redunancy Zones, if an operator chooses to deploy Consul across 3 Availability Zones, they 
+could have 2 (or more) servers (1 voting/1 non-voting) in each zone. In the event that a voting 
+member in an availability zone fails, the Redundancy Zone configuration would automatically 
+promote the non-voting member to a voting member. In the event that an entire Availability 
+Zone was lost, a non-voting member in one of the existing availability zones would promote 
+to a voting member, keeping server quorum. This capability functions as a "hot standby" 
+for server nodes while also providing (and expanding) the capabilities of 
+[Enhanced Read Scalability](/docs/enterprise/read-scale/index.html) by also including recovery
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/redundancy/index.html.md]:
```suggestion
[enhanced read scalability](/docs/enterprise/read-scale/index.html) by also including recovery 
```

File: website/source/docs/enterprise/upgrades/index.html.md [hunk 28]
```diff
@@ -6,14 +6,12 @@ description: |-
...
-[Consul Enterprise](https://www.hashicorp.com/consul.html) supports an [upgrade
-pattern](https://learn.hashicorp.com/consul/day-2-operations/autopilot#upgrade-migrations)
-that allows operators to deploy a complete cluster of new servers and then just wait
-for the upgrade to complete. As the new servers join the cluster, server
-introduction logic checks the version of each Consul server. If the version is
-higher than the version on the current set of voters, it will avoid promoting
-the new servers to voters until the number of new servers matches the number of
-existing servers at the previous version. Once the numbers match, Autopilot will
-begin to promote new servers and demote old ones.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) enables the capability of automatically upgrading a Consul cluster to a new
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/upgrades/index.html.md]:
```suggestion
[Consul Enterprise](https://www.hashicorp.com/consul.html) enables the capability of automatically upgrading a cluster of Consul servers to a new
```
Is it only the servers that are automated or do client upgrades also get automated?

File: website/source/docs/enterprise/upgrades/index.html.md [hunk 29]
```diff
@@ -6,14 +6,12 @@ description: |-
...
+version as updated server nodes join the cluster. This automated upgrade will spawn a process which monitors the amount of voting members 
+currently in a cluster. When an equal amount of new server nodes are joined running the desired version, the lower versioned servers 
+will be demoted to non voting members. Demotion of legacy server nodes will not occur until the voting members on the new version match. 
+Once this demotion occurs, the previous versioned servers can be removed from the cluster safely. 
+
+You can review more information about this functionality in the [Consul Operator Autopilot](https://www.consul.io/docs/commands/operator/autopilot.html) documentation as well as on the HashiCorp Learn [Automated Upgrade](https://learn.hashicorp.com/consul/day-2-operations/autopilot#upgrade-migrations) guide.
```
Comment (kaitlincarter-hc) [inline website/source/docs/enterprise/upgrades/index.html.md]:
```suggestion
You can review more information about this functionality in the [Consul operator autopilot](https://www.consul.io/docs/commands/operator/autopilot.html) documentation as well as on the HashiCorp Learn [Automated Upgrade](https://learn.hashicorp.com/consul/day-2-operations/autopilot#upgrade-migrations) guide.
```

File: website/source/docs/enterprise/index.html.md (hunk 9 above)
Comment (codyde) [reply website/source/docs/enterprise/index.html.md]: Upcoming changes as in future downstream documentation changes? I don't have one immediately. We have one for the future state updating of .io docs in general - but I'll look at creating another one for the next round of changes to include.

File: website/source/docs/enterprise/index.html.md (hunk 10 above)
Comment (codyde) [reply website/source/docs/enterprise/index.html.md]: I had some table ideas that were a bit more involved and I want to see about tackling alongside the .io page updates. I really want to do a table here - just going to be a bit more involved short term.

File: website/source/docs/enterprise/upgrades/index.html.md (hunk 28 above)
Comment (codyde) [reply website/source/docs/enterprise/upgrades/index.html.md]: Correct, only servers. Good call out here!

**Pull request merged**